pkg/collector/corechecks/containers/generic/adapters_utils.go

```diff
@@ -0,0 +1,11 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package generic
+
+// ConvertNanosecondsToHz converts nanoseconds to User Hz (1/100s)
+func ConvertNanosecondsToHz(val float64) float64 {
+	return val / 10e6
+}
```

Comment:
I’m not sure if it’s the function which is badly named or if there is a mistake in the formula but there are several contradictory things here.

`val ÷ 10⁶` actually converts a `val` value in nanoseconds into milliseconds.
A Hertz is `1 s⁻¹` so that converting a period in nanoseconds into a frequency in Hertz is done by `10⁹ ÷ val`.
The comment also mentions hundredths of a second (`¹/₁₀₀ s`) but converting a duration in nanoseconds into hundredths of a second is done with `val ÷ 10⁷`.

Reply:
The notation is misleading, `10e6` = `1e7` = `10^7`, so the function is correct. I changed to `1e7` in further commits to clarify.